tests/packages.js

```diff
@@ -0,0 +1,51 @@
+const { spawn } = require('child_process');
+const path = require('path');
+
+const tests = [
+    ["./packages/packages/as", "node ../../../../bin/asc as/index.ts --noEmit --runtime stub --validate"],
```

Comment:
It would be nice to inline these into the code rather than calling `spawn` I think.